accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/helper/ContentHelper.java

```diff
@@ -137,7 +136,7 @@ public static void importContent(final Session session, final String path,
 
             ImportOptions importOptions = new ImportOptions();
             importOptions.setAutoSaveThreshold(Integer.MAX_VALUE); // IMPORTANT: this disables saving the session in Importer
-            importOptions.setStrict(true);
+            importOptions.setStrict(false);

```

Comment:
@kwin So this is the real change, that makes the difference? (apart from the test and test infra changes). Not using strict is ok for this use case I think